packages/ember-toucan-form/src/components/toucan-form.gts

```diff
@@ -14,6 +14,8 @@ import TextareaFieldComponent from '../-private/textarea-field';
 
 import type { HeadlessFormBlock, UserData } from '../-private/types';
 import type { WithBoundArgs } from '@glint/template';
+import type { TrackedAsyncData } from 'ember-async-data';
```

Comment:
Given we are using this package in client code, should the dependency be under dependencies rather than devDependencies?

(Fair warning: reviewing on mobile so I may be missing something 😅)

Reply:
Brad changed this on my request actually! 😅 See https://github.com/CrowdStrike/ember-toucan-core/pull/466#discussion_r1860113683

Reply:
Ah, makes sense. Thanks Simon! I missed that!